lib/orchestrator_client.rb

```diff
@@ -11,53 +13,45 @@ class OrchestratorClient
   require 'orchestrator_client/config'
 
   attr_accessor :config
+  attr_reader :http
 
   def initialize(overrides, load_files=false)
     @config = Config.new(overrides, load_files)
     @config.validate
+    @http = create_http(config.root_url)
   end
 
   def make_uri(path)
```

Comment:
This function could be removed.

Reply:
I will take care of this in https://github.com/puppetlabs/orchestrator_client-ruby/pull/20

Thanks @seanmil this will go in next release.